src/assets/icons/CheckSuccess.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 56 (81588) - https://sketch.com -->
+    <title>09 Icons / Check / Regular Copy</title>
+    <desc>Created with Sketch.</desc>
+    <g id="09-Icons-/-Check-/-Regular-Copy" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="Check" transform="translate(5.000000, 7.000000)" fill="#57C487" fill-rule="nonzero">
+            <polygon id="Path-2" points="13.3 0 14 0.7 5.3 10 0 5.8 0.6 5 5.2 8.7"></polygon>
+        </g>
+    </g>
+</svg>
```

Comment:
The svg just needs to be cleaned a bit.
Remove id's and unnecessary components (title).
It's just that it causes problems sometimes.